cc/selfplay.cc

```diff
@@ -135,6 +136,12 @@ DEFINE_int32(parallel_games, 32, "Number of games to play in parallel.");
 DEFINE_int32(num_games, 0,
              "Total number of games to play. Defaults to parallel_games. "
              "Only one of num_games and run_forever must be set.");
+DEFINE_int32(cache_size_mb, 0, "Size of the inference cache in MB.");
```

Comment:
if the cache is only on selfplay, it will be hard to ensure that this isn't affecting the playing strength.  Should this flag be moved into the mcts_player?

Reply:
I'm trying to keep flags out of the library build targets. This leads to some copy-paste in the selfplay & eval binaries but it's worth it IMO: https://abseil.io/tips/45

Also, we need to share the inference cache between player instances, so it needs to be created at the top level.